src/org/jitsi/impl/neomedia/rtp/MediaStreamTrackDesc.java

```diff
@@ -127,7 +127,7 @@ public MediaStreamTrackReceiver getMediaStreamTrackReceiver()
      * @param frameDesc
      * @param nowMs
```

Comment:
docs need updating...it'd be nice to have a good definition for what `isNew` really denotes...i'm not sure i know.  is it the first frame after a stream either 1) begins for the first time or 2) resumes after being suspended?